src/pages/docs/test-management/settings/labels.md

```
@@ -0,0 +1,85 @@
+  name: "Delete a Label"
+  url: "#delete-a-label"
+---
+
+<br>
+
+**Use labels to organize test cases and test runs for easier filtering and tracking.**
+
+---
+
+Labels help you organize your test cases and test runs, making it easier to filter and track them. In the Manage Labels section, you can:
+   - Add new labels to categorize tests based on functionality (e.g., Smoke Test, Regression, API Testing).
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)